ComputerVision/Rest/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,154 @@
+                // into a byte array.
+                byte[] byteData = GetImageAsByteArray(imageFilePath);
+
+                // Add the byte array as an octet stream to the request body.
+                using (ByteArrayContent content = new ByteArrayContent(byteData))
+                {
+                    // This example uses the "application/octet-stream" content type.
+                    // The other content types you can use are "application/json" and "multipart/form-data".
+                    content.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
+
+                    // Asynchronously call the REST API method.
+                    response = await client.PostAsync(uri, content);
+                }
+
+                // Asynchronously get the JSON response.
+                string contentString = await response.Content.ReadAsStringAsync();
+
+                // Display the JSON response.
+                Console.WriteLine("\nResponse:\n\n{0}\n", JToken.Parse(contentString).ToString());
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("\n" + e.Message);
+            }
+        }
+        
+        static byte[] GetImageAsByteArray(string imageFilePath)
+        {
+            // Open a read-only file stream for the specified file.
+            using (FileStream fileStream = new FileStream(imageFilePath, FileMode.Open, FileAccess.Read))
+            {
+                // Read the file's contents into a byte array.
+                BinaryReader binaryReader = new BinaryReader(fileStream);
+                return binaryReader.ReadBytes((int)fileStream.Length);
+            }
+        }
+
+        static async Task AnalyzeFromUrlAsync(string imageUrl)
+        {
+            if (!Uri.IsWellFormedUriString(imageUrl, UriKind.Absolute))
+            {
+                Console.WriteLine("\nInvalid remote image url:\n{0} \n", imageUrl);
+                return;
+            }
+
+            try
+            {
+                HttpClient client = new HttpClient();
+
+                // Request headers.
+                client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+                // Request parameters. A third optional parameter is "details".
+                // Comment parameters that aren't required
+                string requestParameters = "visualFeatures=" +
+                    "Categories," +
+                    "Description," +
+                    "Color, " +
+                    "Tags, " +
+                    "Faces, " +
+                    "ImageType, " +
+                    "Adult , " +
+                    "Brands , " +
+                    "Objects"
+                    ;
+
+                //Assemble the URI and content header for the REST API request
+                string uri = uriBase + "?" + requestParameters;
+
+                HttpResponseMessage response;
```

Comment:
Can you move this down below and combine it with the line where it is consumed? 

HttpResponseMessage response = await client.PostAsync(uri, content);